lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -973,8 +973,7 @@
           @include idashbutton($blue);
           position: absolute;
           @include right($baseline);
-        }
-      }
+    }
```

Comment:
Nit: the wrong indentation on the lines above should be fixed too. This diff looks wrong with limited context because of the bad indentation.